src/agent/context.ts

```diff
@@ -26,7 +26,7 @@ function ensureTrace(writer: cm.ITraceWriter) {
 }
 
 export class WellKnownVariables {
-    public static sourceFolder = "build.sourceDirectory";
+    public static sourceFolder = "build.sourcesDirectory";
```

Comment:
We shouldn't set the explicit string here.  Note that common.js (cm module) has the vars defined but we need to expose via context here since tasks and plugins get access to the context.

So, I think this should be 

    public static sourceFolder = cm.vars.buildSourcesDirectory;